platforms/twitter.go

```diff
@@ -1,17 +1,19 @@
 package cmd
 
 import (
+	"log"
+	"os"
+
 	"github.com/dghubble/go-twitter/twitter"
 	"github.com/dghubble/oauth1"
-	"log"
 )
 
-func AuthenticateTwitter() (*twitter.Client) {
+func AuthenticateTwitter() *twitter.Client {
 
-	const ConsumerKey = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-	const ConsumerSecret = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-	const AccessToken = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-	const AccessSecret = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+	ConsumerKey := os.Getenv("CONSUMER_KEY")
+	ConsumerSecret := os.Getenv("CONSUMER_SECRET")
+	AccessToken := os.Getenv("ACCESS_TOKEN")
+	AccessSecret := os.Getenv("ACCESS_TOKEN_SECRET")
 
```

Comment:
Add a check to check if the variables filled from ENV are not empty.

Reply:
` os.LookupEnv` used and checks added. 
![image](https://user-images.githubusercontent.com/67012359/169460044-b1a90285-d9f4-4d62-9f51-06a6101c3a08.png)